src/ObjectCalisthenics/AbstractIdentifierLengthSniff.php

```diff
@@ -42,6 +42,13 @@
      */
     private $stackPtr;
 
+    /**
+     * List of variables names accepted.
+     *
+     * @var array
+     */
+    private $exceptions = ['id', 'em'];
```

Comment:
"em" is shortcut for Event Manger, Entity Manager? If so, should be written in full words. If not, it's confusing as well.
"id" makes sense